lib/phoenix/presence.ex

```diff
@@ -205,6 +210,13 @@ defmodule Phoenix.Presence do
   end
 
   @doc """
+  Returns true if there's a certain presence for a topic.
+  """
+  def includes?(module, topic, key) do
```

Comment:
It would be nice to push this to the tracker. If we are still traversing the whole list, we are not gaining much IMO.